config_src/drivers/nuopc_cap/mom_surface_forcing_nuopc.F90

```diff
@@ -186,6 +195,19 @@ module MOM_surface_forcing_nuopc
                                                               !< on ocean surface [Pa]
   real, pointer, dimension(:,:) :: ice_fraction      =>NULL() !< fractional ice area [nondim]
   real, pointer, dimension(:,:) :: u10_sqr           =>NULL() !< wind speed squared at 10m [m2/s2]
+  real, pointer, dimension(:,:) :: nhx_dep           =>NULL() !< Nitrogen deposition [kg/m^2/s]
+  real, pointer, dimension(:,:) :: noy_dep           =>NULL() !< Nitrogen deposition [kg/m^2/s]
+  real, pointer, dimension(:,:) :: atm_co2_prog      =>NULL() !< Prognostic atmospheric co2 concentration [ppm]
+  real, pointer, dimension(:,:) :: atm_co2_diag      =>NULL() !< Diagnostic atmospheric co2 concentration [ppm]
+  real, pointer, dimension(:,:) :: atm_fine_dust_flux   =>NULL() !< Fine dust flux from atmosphere [kg/m^2/s]
+  real, pointer, dimension(:,:) :: atm_coarse_dust_flux =>NULL() !< Coarse dust flux from atmosphere [kg/m^2/s]
+  real, pointer, dimension(:,:) :: seaice_dust_flux     =>NULL() !< Dust flux from seaice [kg/m^2/s]
+  real, pointer, dimension(:,:) :: atm_bc_flux          =>NULL() !< Black carbon flux from atmosphere [kg/m^2/s]
+  real, pointer, dimension(:,:) :: seaice_bc_flux       =>NULL() !< Black carbon flux from seaice [kg/m^2/s]
+  real, pointer, dimension(:,:) :: afracr               =>NULL()
```

Comment:
It is a very good idea for all real variables to be described in a comment, along with their units and perhaps the sign convention that they use, when they are first defined.  Elsewhere in MOM6, it is actually an automatically enforced requirement that all elements of a transparent public type (as is the case here) are described in a doxygen comment. Please add comments describing the new `afracr`, `swnet_afracr`, `swpen_ifrac_n` and `ifrac_n` elements of the NUOPC version of the `ice_ocean_boundary_type`.

Reply:
Addressed in 5402ce0